oci/config/convert/runtime.go

```diff
@@ -60,29 +61,25 @@ func parseEnv(env string) (string, string, error) {
 // MutateRuntimeSpec mutates a given runtime specification generator with the
 // image configuration provided. It returns the original generator, and does
 // not modify any fields directly (to allow for chaining).
-//
-// XXX: This conversion is not actually defined by the image-spec. There is a
-//      proposal in-the-works though. https://github.com/opencontainers/image-spec/pull/492
-func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image, manifest ispec.Manifest) error {
+func MutateRuntimeSpec(g rgen.Generator, rootfs string, image ispec.Image) error {
 	if image.OS != "linux" {
 		return errors.Errorf("unsupported OS: %s", image.OS)
 	}
 
 	// FIXME: We need to figure out if we're modifying an incompatible runtime spec.
 	//g.SetVersion(rspec.Version)
-	g.SetPlatformOS(image.OS)
-	g.SetPlatformArch(image.Architecture)
 
+	// Set verbatim fields
+	g.SetPlatformArch(image.Architecture)
+	g.SetPlatformOS(image.OS)
 	g.SetProcessTerminal(true)
 	g.SetRootPath(filepath.Base(rootfs))
 	g.SetRootReadonly(false)
 
-	g.SetProcessCwd("/")
```

Comment:
I removed the line, since it's already the default in the generator. Adding such information to the spec wouldn't hurt. I believe that any additional information helps to make the hard requirements and the corresponding degree of freedom more explicit.